### PR TITLE
Storage rebalance

### PR DESCRIPTION
**Pull request by MoXuT0**
## About The Pull Request

Per title. If this shit look unbalanced, that's because it is.

## Changelog

:cl:
balance: Belts. Increased shotgun belt capacity up to 14. Increased regular grenade belt capacity up to 9. Increased pistol and revolver belt capacity up to 7.
balance: Pouches. Increased flare pouch and M40 flare box capacity up to 7. Increased electronics pouch capacity up to 6.
balance: Misc. M37 aka shotgun holster can now hold scout's shotgun. Increased integrated storage armor by 1 (It's now same as satchel). Increased incendiary grenade box nade count up to 25. Increased shotgun box weight class, it can't fit in satchels or backpacks now. Reduced backpack storage by 6, it can now contain 8 normal sized items instead of 10.
balance: Cargo armaments vendor. Added M37 holster, satchel, utility, knife and grenade belts, large general and magazine pouches. Added flamethrower, 88 Mod 4 and KSG shotgun to contraband. Increased ammount of existing items. Removed impact nades.
balance: Cargo munition vendor. Added hollowpoint revolver bullets. Increased amount of some existing items. Added ammo for flamer and 88 Mod 4 to contraband. Moved machete holster to armaments vendor.
balance: Marine vendors . Removed explosive and G8 pouches from regular PFC vendor. Removed machete holster, toolbelt and G8 pouch from uniform vendor, tools pouch now starts empty.
imageadd: Field utility, explosive and shotgun shells pouches now have unique sprites. Recolored syringe pouch.
/:cl:

### CODE REVIEW

**Comment by DominikPanic** [comment]
The grammar changes don't need to be changeloged. This changelog is already huge.

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -211,7 +211,7 @@
 	use_to_pickup = 1
 	storage_slots = null
 	use_sound = null
-	max_storage_space = 16
+	max_storage_space = 12
```
Comment (spookydonut) [inline code/game/objects/items/storage/firstaid.dm]: change this back, maths is better for pill batching with 16

**Comment by MoXuT0** [comment]
Reverted back pills and splitted typos fix in different pr.

File: code/game/objects/items/storage/firstaid.dm (hunk 1 above)
Comment (Rohesie) [reply code/game/objects/items/storage/firstaid.dm]: 8 or 16, agreed.

File: code/datums/gamemodes/distress.dm [hunk 2]
```diff
@@ -387,17 +387,17 @@
...
 						/obj/item/ammobox/m4a3 = round(scale * 3),
+						/obj/item/ammo_magazine/pistol = round(scale * 20),
+						/obj/item/ammobox/m4a3ap = round(scale * 1),
 						/obj/item/ammo_magazine/pistol/ap = round(scale * 5),
-						/obj/item/ammobox/m4a3ap = round(scale * 3),
-						/obj/item/ammo_magazine/pistol/incendiary = round(scale * 2),
+						/obj/item/ammobox/m4a3ext = round(scale * 1),
 						/obj/item/ammo_magazine/pistol/extended = round(scale * 10),
-						/obj/item/ammobox/m4a3ext = round(scale * 3),
+						/obj/item/ammo_magazine/pistol/incendiary = round(scale * 5),
```
Comment (Rohesie) [inline code/datums/gamemodes/distress.dm]: This is a lot of incendiary. Will have to be tested for impact.

File: code/datums/gamemodes/distress.dm [hunk 3]
```diff
@@ -456,49 +440,62 @@
...
+						/obj/item/explosive/grenade/cloakbomb = round(scale * 4),
+						/obj/item/storage/box/nade_box = round(scale * 2),
 						/obj/item/storage/box/m94 = round(scale * 30),
 						/obj/item/flashlight/combat = round(scale * 5),
 						/obj/item/clothing/mask/gas = round(scale * 10)
 						)
 
 		CG.contraband = list(
-						/obj/item/weapon/gun/smg/ppsh = round(scale * 4),
+						/obj/item/storage/box/nade_box/impact = round(scale * 1),
```
Comment (Rohesie) [inline code/datums/gamemodes/distress.dm]: Remove these completely. Only make them orderable.
Consider that contraband will always be done if there's interesting things to get there.
We restrict them for now to meme ineffective stuff, like the PPSH.

**Comment by MoXuT0** [comment]
Mrs @Rohesie I've done it. I've stopped impact nades. Xenos are free to roam this earth.

**Comment by Rohesie** [comment]
We'll see how this goes.

**Pull request merged**